MvvmCross/Windows/Uwp/Views/MvxWindowsViewPresenter.cs

```diff
@@ -28,34 +33,111 @@ public MvxWindowsViewPresenter(IMvxWindowsFrame rootFrame)
             SystemNavigationManager.GetForCurrentView().BackRequested += BackButtonOnBackRequested;
         }
 
-        private IMvxViewModelTypeFinder _viewModelTypeFinder;
-        public IMvxViewModelTypeFinder ViewModelTypeFinder
+        public override void RegisterAttributeTypes()
+        {
+            AttributeTypesToActionsDictionary.Add(
+                typeof(MvxPagePresentationAttribute),
+                new MvxPresentationAttributeAction
+                {
+                    ShowAction = (view, attribute, request) => ShowPage(request),
```

Comment:
It's probably best to send the view, attribute and request parameters already to this method. You might not need them now, but maybe later, or someone overriding this needs it.